src/components/userProfile/UserProfile.js

```diff
@@ -159,39 +160,27 @@ class UserProfile extends Component {
 
     if (this.state.statusAsMentee === ConnectStatus.CONNECTED
         || this.state.statusAsMentor === ConnectStatus.CONNECTED) {
-      connectButton = (
-          <LinearGradient style={styles.connectBtnStyle} start={[0.9, 0.5]} end={[0.0, 0.5]}
-              locations={[0, 0.75]}
-              colors={['#07e4dd', '#44acff']}>
-          <TouchableHighlight>
-            <Text style={styles.buttonText}>WAITING...</Text>
-          </TouchableHighlight>
-          </LinearGradient>
-        );
+      connectBtnText = 'WAITING...';
```

Comment:
Our mentor list updated as list of non-connected user, so maybe now front don't need to check this. FYI